nbpresent/install.py

```diff
@@ -1,25 +1,21 @@
 #!/usr/bin/env python
 
 import argparse
+import os
 from os.path import dirname, abspath, join
 
 from notebook.nbextensions import install_nbextension
 from notebook.services.config import ConfigManager
 
 
-def install(user=False, symlink=False, overwrite=False, enable=False,
-            **kwargs):
+def install(conda_env=False, enable=False, **kwargs):
     """Install the nbpresent nbextension assets and optionally enables the
        nbextension and server extension for every run.
 
     Parameters
     ----------
-    user: bool
-        Install for current user instead of system-wide.
-    symlink: bool
-        Symlink instead of copy (for development).
-    overwrite: bool
-        Overwrite previously-installed files for this extension
+    conda_env: bool
```

Comment:
I don't know if `conda_env` is the best name (because we have something called `conda env`) but I am OK with that for now...